### PR TITLE
openvpn: T2146: remove old files

### PR DESCRIPTION
**Pull request by jjakob**

### CODE REVIEW

**Review by dmbaturin — APPROVED** [review]

**Review by dmbaturin — APPROVED** [review]

File: src/conf_mode/interfaces-openvpn.py [hunk 1]
```diff
@@ -899,6 +899,13 @@ def generate(openvpn):
     interface = openvpn['intf']
     directory = os.path.dirname(get_config_name(interface))
 
+    # we can't know which clients were deleted, remove all client configs
+    if os.path.isdir(directory + '/ccd/' + openvpn['intf']):
```
Comment (thomas-mangin) [inline src/conf_mode/interfaces-openvpn.py]: This works but a better way would be to `os.path.join(directory,'ccd',openvpn['intf'])`

**Comment by c-po** [comment]
Looks good to me, too - is it still a WIP?

File: src/conf_mode/interfaces-openvpn.py (hunk 1 above)
Comment (jjakob) [reply src/conf_mode/interfaces-openvpn.py]: Fixed

**Comment by jjakob** [comment]
Ready to merge.

**Comment by jjakob** [comment]
@Elbandi Indeed, I didn't think of that at the time. It would require more code to list all the files in the directory, find which ones aren't configured clients, and delete just those. The script recreates all the ccd files from scratch soon after so it's not a problem just to delete them all. The server won't read them until being restarted anyway.

**Pull request merged**